include/spirv-tools/libspirv.h

```diff
@@ -300,6 +300,10 @@ typedef enum spv_operand_type_t {
   SPV_OPERAND_TYPE_COOPERATIVE_MATRIX_LAYOUT,
   SPV_OPERAND_TYPE_COOPERATIVE_MATRIX_USE,
 
+  // SPV_ARM_tensors
```

Comment:
Please put new enums at the end of the list.
That gives more chance to be binary compatible if client and implementation use different versions of this header.

Reply:
I have done a rebase of this PR on top of latest main (as of a few minutes ago).

https://github.com/dneto0/SPIRV-Tools/tree/arm-tensors-rebased